src/block/MonsterSpawner.php

```diff
@@ -42,11 +40,4 @@ protected function getXpDropAmount() : int{
 	public function onScheduledUpdate() : void{
 		//TODO
 	}
-
-	public function getSupportType(int $facing) : SupportType{
-		if($facing === Facing::UP || $facing === Facing::DOWN){
-			return SupportType::CENTER();
-		}
-		return SupportType::FULL();
-	}
```

Comment:
This should have NONE support type

Reply:
Indeed, I thought ``Block`` was still setting ``NONE`` by default, so much for me